tests/foreman/cli/test_setting.py

```diff
@@ -285,6 +286,50 @@ def test_positive_update_send_welcome_email(self):
               {'search': 'name=send_welcome_email'})[0]['value']
             assert value == host_value
 
+    @tier1
+    def test_positive_enable_disable_rssfeed(self):
+        """Check if the RSS feed can be enabled or disabled
+
+        :id: 021cefab-2629-44e2-a30d-49c944d0a234
+
+        :steps: Set rss_enable true or false
+
+        :expectedresults: rss_enable is updated
+
+        :caseautomation: automated
+        """
+        for value in ['true', 'false']:
+            Settings.set({'name': 'rss_enable', 'value': value})
+            rss_setting = Settings.list({'search': 'name=rss_enable'})[0]
+            self.assertEqual(value, rss_setting['value'])
```

Comment:
should we record and restore the default value here? It's done in the second test so if there is a risk of interfering with other tests, I'd do it also here

Reply:
Ok, good point, I have missed this. Added.